thunder/__init__.py

```diff
@@ -796,10 +798,10 @@ def last_interpreted_instructions(fn: Callable) -> list[dis.Instruction]:
         raise TypeError(f"{fn} doesn't seem to be a thunder compiled function.")
     if cs.last_interpreted_instructions is None:
         raise TypeError(f"{fn} doesn't seem to have been called yet.")
-    return cs.last_interpreted_instructions
+    return list(cs.last_interpreted_instructions)
 
 
-def last_interpreted_history(fn: Callable) -> list[dis.Instruction | str]:
+def last_interpreted_history(fn: Callable) -> list[InterpreterHistoryItem]:
```

Comment:
I wonder if we should use the opportunity to call it a `interpretation_log` rather than a `interpreted_history`. For one, history is very generic, also it's not interpreted but it recorded during interpretation.

Reply:
Agreed. Could also call it `interpreter_log`. What do you think?

Also, it would have to be fixed everywhere we call it history, which is a lot of places. But it's greppable, so not that big a deal.

Reply:
Seems good, thank you!